core/test/element_math_test.cpp

```diff
@@ -46,6 +47,28 @@ TEST(ElementNormTest, value) {
   EXPECT_EQ(element::norm(v2), 5);
 }
 
+TEST(ElementPowTest, unit) {
+  // element::pow cannot handle units itself as that requires the unit of the
```

Comment:
This comment doesn't read properly

Reply:
Then I need help from a native speaker, it sounds fine to me.

Reply:
Current: ```element::pow cannot handle units itself as that requires the unit of the base but the *value* of the exponent. This does not fit into the usual```

Suggested: ```element::pow cannot handle units itself as that requires the  the *value* of the exponent not the unit of the base. This does not fit into the usual```

Reply:
Did you mean 'value of the exponent and not its unit'? As it stands, your suggestion is incorrect.